aiida/orm/nodes/data/code/abstract.py

```diff
@@ -59,6 +61,7 @@ def __init__(
         :param append_text: The text that should be appended to the run line in the job script.
         :param prepend_text: The text that should be prepended to the run line in the job script.
         :param use_double_quotes: Whether the command line invocation of this code should be escaped with double quotes.
+        :param with_mpi: Whether the command should be run with the MPI run command in front.
```

Comment:
I am fine with changing it to "MPI program" but I think the language should include that it forces something. Your suggested phrasing seems to suggest it is just a label but it is not clear whether it will change behavior. How about "Whether the code should be run as an MPI program."

Reply:
Sounds good!